config/environments/development.rb

```diff
@@ -11,6 +11,9 @@
 
   # Show full error reports.
   config.consider_all_requests_local = true
+  
+  # Enable serving static assets
+  config.public_file_server.enable
```

Comment:
Per the the latest rails docs: [railsguides](http://edgeguides.rubyonrails.org/configuring.html)

config.public_file_server.enabled configures Rails to serve static files from the public directory. This option defaults to true, but in the production environment it is set to false because the server software (e.g. NGINX or Apache) used to run the application should serve static files instead. If you are running or testing your app in production mode using WEBrick (it is not recommended to use WEBrick in production) set the option to true. Otherwise, you won't be able to use page caching and request for files that exist under the public directory.

@techpeace Are you sure we need this?